docs/src/examples/components/TextArea/Usage/TextAreaHeightExample.shorthand.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react'
+import { TextArea } from '@stardust-ui/react'
+
+const TextAreaMaxLengthExample = () => (
+  <TextArea placeholder="Enter message here..." variables={{ height: '200px' }} />
+)
+
+export default TextAreaMaxLengthExample
```

Comment:
```suggestion
export default TextAreaExampleHeight
```

Filename should be also changed. We can also use `useRangeKnob()` for this example, see `docs/src/examples/components/Chat/Usage/ChatExampleInScrollable.shorthand.tsx`